include/cpu_features_macros.h

```diff
@@ -231,6 +247,84 @@
 #endif
 #endif  //  defined(CPU_FEATURES_ARCH_MIPS)
 
+#if defined(CPU_FEATURES_ARCH_RISCV)
+#if defined(__riscv_m)
+#define CPU_FEATURES_COMPILED_RISCV_M 1
```

Comment:
what about `__riscv_i`, `__riscv_`, `__riscv_p`, `__riscv_zfh`?
ref: https://github.com/riscv-non-isa/riscv-c-api-doc/blob/master/riscv-c-api.md

Reply:
Thanks for the reply!

My reasoning got the omissions (which might not be good reasons) was
- I have some doubt as to whether RV32E embedded processors are capable enough to run the cpu_features library
- The p extension isn't ratified yet (and might never be)
- Zfh currently isn't supported by QEMU (although someone has submitted a patch), so although I can test compiler support, the next step (checking hwcap to see if the cpu you're running on supports it) is going to be difficult to test

i'll update this patch to add I, e, zfh and zfhmin. As p isn't ratified, I'd rather leave it out unless someone really wants it.

Reply:
Do you know which version of QEMU is needed ? so we can bump
https://github.com/google/cpu_features/blob/3c4801d12d9bec823d03a572f4f53cd42fa02b42/scripts/run_integration.sh#L36